### PR TITLE
add acc contributor template with no sgx package install

### PR DESCRIPTION
**Pull request by BRMcLaren**
Template for setting up automated Linux regression testing for Intel Packages. Allow installation of all dependencies but not including the SGX packages.

### CODE REVIEW

**Comment by BRMcLaren** [comment]
bors try

**Comment by jhand2** [comment]
I don't really understand what this is for. Does "no sgx" mean "no psw"? Because it looks like the sgx driver is still installed.

**Comment by BRMcLaren** [comment]
@jhand2 this is correct. It's a template that installs everything except the SGX packages, but including the driver.

The use case is to allow us to test against the Intel release candidate packages by setting up a machine with all the dependencies but the SGX packages.  Open to other suggestions, but this is what we have been doing manually up until now.

**Comment by jhand2** [comment]
> @jhand2 this is correct. It's a template that installs everything except the SGX packages, but including the driver.
>
> The use case is to allow us to test against the Intel release candidate packages by setting up a machine with all the dependencies but the SGX packages. Open to other suggestions, but this is what we have been doing manually up until now.

That seems fine to me. My only suggestion would be to change the name to `oe-contributors-acc-setup-no-psw.yml` for clarity

**Comment by BRMcLaren** [comment]
Sure thing.

Changed the file name for clarity.

**Comment by BRMcLaren** [comment]
bors try

File: scripts/ansible/oe-contributors-acc-setup-no-psw.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            name: linux/openenclave
+            tasks_from: environment-setup.yml
+
+        - import_role:
+            name: linux/intel
+            tasks_from: sgx-driver.yml
+
+        - import_role:
+            name: linux/az-dcap-client
+            tasks_from: stable-install.yml
```
Comment (CodeMonkeyLeet) [inline scripts/ansible/oe-contributors-acc-setup-no-psw.yml]: Nit: missing newline

Comment (BRMcLaren) [reply scripts/ansible/oe-contributors-acc-setup-no-psw.yml]: Fixed.

**Pull request merged**